### PR TITLE
[Doc] Fix typo in md files

### PR DESCRIPTION
**Pull request by kimjh12**
Typo correction in md files.

Signed-off-by: Junhwan Kim <jejudo.kim@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

**Review by zhoonit — APPROVED** [review]
Nice catch :)

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Contributing
...
+The initial TSC voting members are the committers of NNStreamer at the time of the first TSC meeting. Election or removal of TSC voting members should be approved by TSC voting. Except for approvals of license exceptions and the community charger amendments, approval of TSC voting requires approvals of more than half of present TSC voting members in a TSC meeting, which requires to have at least the half of the voting members present in the TSC meeting. Approvals of electronic TSC voting requires approvals of more than half of all TSC voting members. TSC may elect or replace the chairperson of the project with TSC voting.
```
Comment (wooksong) [inline CONTRIBUTING.md:9]:
```suggestion
The initial TSC voting members are the committers of NNStreamer at the time of the first TSC meeting. Election or removal of TSC voting members should be approved by TSC voting. Except for approvals of license exceptions and the community charger amendments, approval of TSC voting requires approvals of more than half of present TSC voting members in a TSC meeting, which requires to have at least the half of the voting members present in the TSC meeting. Approvals of electronic TSC voting require approvals of more than half of all TSC voting members. TSC may elect or replace the chairperson of the project with TSC voting.
```

Comment (wooksong) [reply CONTRIBUTING.md:9]: Already merged:(

Comment (jaeyun-jung) [reply CONTRIBUTING.md:9]: @kimjh12 plz add another PR with wook's comment TT

Comment (wooksong) [reply CONTRIBUTING.md:9]: @jaeyun-jung @kimjh12 lol. Never mind. The comment was very trivial.

**Pull request merged**